.github/workflows/verify-notebooks.yml

```diff
@@ -0,0 +1,38 @@
+name: Verify Notebooks
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - ".github/workflows/verify-notebooks.yml"
+      - "examples/model-deployment/"
+      - "text-image-search/"
+  pull_request:
+    branches:
+      - master
+    paths:
+      - ".github/workflows/verify-notebooks.yml"
+      - "examples/model-deployment/"
+      - "text-image-search/"
+
+defaults:
+  run:
+    # Specify to ensure "pipefail and errexit" are set.
+    # Ref: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#defaultsrunshell
+    shell: bash
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          pip install --upgrade -qqq pytest notebook nbconvert runnb pandas torch numpy scikit-learn
```

Comment:
runnb is obscure package, better to use something better known and supported.
For pyvespa, we use papermill to run notebooks:
https://github.com/vespa-engine/pyvespa/blob/master/.github/workflows/notebooks-cloud.yml

Reply:
I don't know neither of the two, so my opinion is void. 😊 

@glebashnik perhaps make it a ticket in JIRA so it can be evaulated?